app/models/operatingsystem.rb

```diff
@@ -193,19 +193,43 @@ def supports_image
     false
   end
 
-  # override in sub operatingsystem classes as required.
-  def pxe_variant
-    "syslinux"
-  end
-
-  # The kind of PXE configuration template used. PXELinux and PXEGrub are currently supported
-  def template_kind
-    "PXELinux"
+  # Compatible kinds for this OS sorted in preferred way
```

Comment:
Mind to move the PXE relevant logic to a concern? It doesn't have much to do with the OS ActiveRecord model itself, it's applicable only under certain conditions too